packages/components/src/internal/components/base/Grid.tsx

```diff
@@ -255,18 +255,10 @@ interface GridBodyProps {
     isLoading: boolean;
     loadingText: ReactNode;
     rowKey: any;
-    transpose: boolean;
```

Comment:
Looks like the transpose prop has been removed—I might edit the comment `// unique key to add to QuerySelect to avoid duplication w/ transpose` in QueryFormInputs.tsx so it's not confusing to future devs